src/nextjs/package.json

```diff
@@ -9,13 +9,13 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@line/liff": "2.22.2",
-    "next": "11.1.3",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "@line/liff": "2.24.0",
+    "next": "15.0.1",
+    "react": "18.3.1",
+    "react-dom": "18.3.1"
   },
   "devDependencies": {
-    "eslint": "7.30.0",
-    "eslint-config-next": "11.0.1"
+    "eslint": "9.13.0",
+    "eslint-config-next": "15.0.1"
```

Comment:
The following error occurs when lint is run.

```
$ yarn run lint
yarn run v1.22.21
$ next lint
Failed to load config "next/babel" to extend from.
Referenced from: /path/to/line/line-liff-v2-starter/src/nextjs/.eslintrc
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Could you modify the ESLint configuration file with reference to the current Next.js template?
https://github.com/vercel/next.js/blob/v15.0.1/packages/create-next-app/templates/default/js/eslintrc.json

Reply:
I have fixed it. 4191fff